site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -178,5 +178,43 @@ pipeline:
         tier: mysql
 ```
 
+### `selectors`
+
+Selectors can be used to target specific resources for function execution. Selectors 
+follow OR of AND(s) approach where, within each selector, the filters are ANDed 
```

Comment:
Before devling into details of boolean logic, need to provide more context of what problem they solve.

Reply:
Also a common misconception with this feature is conflating it with filtering. Need to explicitly address in the example. A target selector does not affect subsequent functions.

Reply:
Yes, lets start with describing a few use-cases first where users will need the capability to target resources and then describe the feature and it's details. 

@phanimarupaka Initially we were worried that the content will be very lengthy and might disrupt the book flow, but looking at the content in the guide, we should be able to cover the details here instead of the guide. WDYT ?